protos/dcs/common/v0/common.proto

```diff
@@ -380,13 +375,52 @@ message MarkPanel {
 }
 
 /**
- * A vector in a left-handed coordinate system with +z being north, -z south, +x
- * east, -x west, +y up and -y down (as opposed to DCS' unusual right-handed
- * coordinate system where x is north/ south and z is west/east; the underlying
- * conversion is basically swapping `x` and `z` as in `x=z` and `z=x`).
+ * A vector in a right-handed coordinate system where +x is north, -x south, +z is east, -z west,
+ * +y up and -y down.
  */
 message Vector {
   double x = 1;
   double y = 2;
   double z = 3;
 }
+
+/**
+  * Information about an object in 3D space, including its position,
+  * orientation and velocity.
+  */
+message Transform {
+  dcs.common.v0.Position position = 1;
+  // Distance between DCS' map origin to object in meters on west-east axis.
+  double u = 2;
```

Comment:
While I know this was requested I don't actually think it is useful information. I have never seen anyone figure out how to convert the internal coord system to lat/long outside of the DCS lua environment since some of the maps are skewed.

I think it was @justin-lovell who requested this. @justin-lovell, What is the use-case?

Reply:
I need them when exporting units into a [ACMI (TacView) format](https://www.tacview.net/documentation/acmi/en/). Since DCS is basically just a flat world, I had way better results when using `u`/`v` compared to just using lat/lon.